index.js

```diff
@@ -0,0 +1,30 @@
+const taskInput = document.getElementById('myInput');
+const taskList = document.getElementById('taskList');
+
+taskInput.addEventListener('keydown', function (event) {
```

Comment:
이벤트 핸들러가 함수 선언식으로 작성되었는데 요즘은 화살표 함수를 많이 쓰는 편이에요~

```suggestion
taskInput.addEventListener('keydown', (event) => {
```

Reply:
이벤트 핸들러 함수 자체를 따로 분리해도 좋아요.

```suggestion
taskInput.addEventListener('keydown', handleKeydownTaskInput);
```

Reply:
수정했습니다~

Reply:
첫번째 댓글에서는 헷갈렸는데 `function (event) { ... }` 요거는 함수 표현식으로 보는게 맞을 것 같아요~